bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractBaseImpl.java

```diff
@@ -155,6 +158,11 @@ public boolean isTooltipVisible() {
         return tooltipVisible;
     }
 
+    @Override
+    public String getLang() {
+        return lang;
```

Comment:
we are returning lang for every field then.

Reply:
yes, but only if it is set, it would be returned.